zcash_client_sqlite/src/wallet/init/migrations/v_transactions_transparent_history.rs

```diff
@@ -134,6 +136,8 @@ impl RusqliteMigration for Migration {
             GROUP BY notes.account_id, notes.txid;"
         )?;
 
+        // FIXME: This *will* include outputs that are in the utxos table with unknown height.
+        // Is that correct?
```

Comment:
I think this is correct, as is the view. We don't expose `utxos.height` here (so there is no need to coalesce `-1` and `NULL`), and we want to include all outputs of all transactions, regardless of the transaction state.